Source/WebKit/WebKit.xcodeproj/project.pbxproj

```diff
@@ -13809,16 +13823,6 @@
 			path = Automation;
 			sourceTree = "<group>";
 		};
-		99A5149C2D5C0CA000937177 /* protocol */ = {
-			isa = PBXGroup;
-			children = (
-				99A514992D5C0CA000937177 /* BidiBrowser.json */,
-				99A5149A2D5C0CA000937177 /* BidiBrowsingContext.json */,
-				99A5149D2D5C0CF600937177 /* BidiLog.json */,
-			);
-			path = protocol;
-			sourceTree = "<group>";
-		};
```

Comment:
What's going on here?

Reply:
I have no idea, Xcode wanted this change. I can put it back if you want?

Reply:
Maybe someone deleted the files and didn't remove from Xcode. Fine to keep if so.